plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -213,12 +213,8 @@ export class PersonState {
             )
         }
 
-        const update: Partial<Person> = {}
-        const updatedProperties = this.updatedPersonProperties(personFound.properties || {})
+        const update: Partial<Person> = this.updatedPersonProperties(personFound)
 
-        if (!equal(personFound.properties, updatedProperties)) {
```

Comment:
This will increase load on ingestion significantly - e.g. due to geoip plugin.

We can always reconstruct the "history" from event list - increasing load for _most_ events ingested is just not worth it for 0 behavioral changes for the end user.

Reply:
yup, I was thinking the same, until we use the timestamp, we shouldn't trigger updates just for that. The PR was still in draft mode because I wasn't finished yet as you can see from tests failing, thanks for the review though.

Reply:
you did tag me for a review :sweat_smile:

Reply:
yeah sorry 😅